src/component/Tooltip.tsx

```diff
@@ -90,99 +99,97 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Def
   useTranslate3d?: boolean;
 };
 
-export class Tooltip<TValue extends ValueType, TName extends NameType> extends PureComponent<
-  TooltipProps<TValue, TName>
-> {
-  static displayName = 'Tooltip';
-
-  static defaultProps = {
-    active: false,
-    allowEscapeViewBox: { x: false, y: false },
-    reverseDirection: { x: false, y: false },
-    offset: 10,
-    viewBox: { x1: 0, x2: 0, y1: 0, y2: 0 },
-    coordinate: { x: 0, y: 0 },
-    cursorStyle: {},
-    separator: ' : ',
-    wrapperStyle: {},
-    contentStyle: {},
-    itemStyle: {},
-    labelStyle: {},
-    cursor: true,
-    trigger: 'hover',
-    isAnimationActive: !Global.isSsr,
-    animationEasing: 'ease',
-    animationDuration: 400,
-    filterNull: true,
-    useTranslate3d: false,
-  };
-
-  state = {
-    boxWidth: -1,
-    boxHeight: -1,
-    dismissed: false,
-    dismissedAtCoordinate: { x: 0, y: 0 },
-  };
-
-  private wrapperNode: HTMLDivElement;
-
-  componentDidMount() {
-    this.updateBBox();
-  }
-
-  componentWillUnmount() {
-    document.removeEventListener('keydown', this.handleKeyDown);
-  }
-
-  componentDidUpdate() {
-    this.updateBBox();
-  }
-
-  handleKeyDown = (event: KeyboardEvent) => {
-    if (event.key === 'Escape') {
-      this.setState({
-        dismissed: true,
-        dismissedAtCoordinate: {
-          ...this.state.dismissedAtCoordinate,
-          x: this.props.coordinate.x,
-          y: this.props.coordinate.y,
-        },
-      });
-    }
-  };
+const tooltipDefaultProps = {
+  active: false,
+  allowEscapeViewBox: { x: false, y: false },
+  reverseDirection: { x: false, y: false },
+  offset: 10,
+  viewBox: { x: 100, y: 100, height: 50, width: 50 },
+  coordinate: { x: 0, y: 0 },
+  wrapperStyle: {},
+  // this doesn't exist on TooltipProps
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  cursorStyle: {},
+  separator: ' : ',
+  contentStyle: {},
+  itemStyle: {},
+  labelStyle: {},
+  isAnimationActive: !Global.isSsr,
+  animationEasing: 'ease',
+  animationDuration: 400,
+  filterNull: true,
+  useTranslate3d: false,
+  cursor: true,
+  trigger: 'hover',
```

Comment:
Nit: Any reason these a ordered differently now? If you want to change the order, alphabetically might be nice. Else keeping a PR as small as possible (i.e. no redundant changes) would set a great example.

Reply:
No reason, just how it happened. Will move back to original order